addons/sale_loyalty/tests/test_program_numbers.py

```diff
@@ -773,9 +727,10 @@ def test_coupon_and_coupon_discount_fixed_amount_tax_excl(self):
         coupon = coupon_program.coupon_ids
         self._apply_promo_code(order, coupon.code)
         self._auto_rewards(order, self.all_programs)
-        self.assertEqual(order.amount_tax, 13.5)
+        self.assertEqual(order.amount_tax, 0.0)
```

Comment:
I'm not sure to understand this test :sweat_smile:

Reply:
TBH, I reverted the changes from the previous commit, that just changed the end values without checking the test still makes sens. Basically, applying 2 different kinds of reward, order should go below 0 with both of it, we check it isn't the case. So, both fixed and percent discount can be applied, and it can't make the order below 0. So, I go back to what this test was created for in the first place, either it's a good test or not is another question completely xD